packages/kit/src/runtime/server/respond.js

```diff
@@ -360,6 +360,9 @@ export async function respond(request, options, manifest, state) {
 			}
 
 			if (route) {
+				const page_methods = ['GET', 'HEAD', 'POST'];
```

Comment:
this array could be created at the top of the file so that it doesn't need to be recreated each time the method runs